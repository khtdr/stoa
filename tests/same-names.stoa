#!/usr/bin/env stoa

var a = "a";
var a = "A";

{
  var a = "Eh.";
  var b = "Be";
  var b = "bee";
}
