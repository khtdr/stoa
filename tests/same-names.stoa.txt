Parse error in ./same-names.stoa at line,col 10,7
10.   var b = "bee";
          â†‘
Variable is already defined
Parse error in ./same-names.stoa at line,col 16,7
16.   var c = "Sea";
          â†‘
Variable is already defined
