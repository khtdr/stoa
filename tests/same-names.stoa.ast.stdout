(program 
   (var a a)
   (var a A)
   (block 
      (var a Eh.)
      (var b Be)
      (var b bee)
   )
   (block 
      (var a eh...)
      (var c Si)
      (var c Sea)
   )
)
